docs/api-reference/legacy.rst

```diff
@@ -84,31 +63,26 @@ The Simple API implements the HTML-based package index API as specified in `PEP
 
     .. code:: http
 
-        GET /simple/warehouse/ HTTP/1.1
+        GET /simple/beautifulsoup/ HTTP/1.1
         Host: pypi.org
         Accept: text/html
 
     **Example response**:
 
     .. code:: http
 
-        HTTP/1.0 200 OK
-        Content-Type: text/html; charset=utf-8
-        X-PyPI-Last-Serial: 867465
-
```

Comment:
I think these lines should probably stay? You'll want to update the serial number to match the one in the comment below.

Reply:
Thank you, @di, I put those lines back. I am not sure if I put in the correct serial number.

Reply:
The serial number can change at any time, so for this example it really can be anything as long as it's a) present and b) consistent.